sln_changes.txt

```diff
@@ -0,0 +1,22 @@
+IAccessibleDLL
```

Comment:
Can you explain why this file is needed exactly? Is it for development purposes only? Because library end-user should not need these notes or am I wrong?

Reply:
This file is not needed, only added to clarify what I had changed in the solution files. I'll go ahead and remove it - but I can add it to a changelog of sorts if you think this information is useful to keep